app/services/applier/gh_form_filler.rb

```diff
@@ -6,11 +6,6 @@ class GhFormFiller < FormFiller
 
     def application_form = '#application_form'
 
-    def click_submit_button
-      sleep 2
-      p "I didn't actually submit the application."
-    end
-
     def attach_file_to_application
```

Comment:
Similarly adding comments to all individual formfillers - particularly can we highlight any special cases / exceptions for why functionality needs to sit here

Reply:
Would prefer to do this incrementally if that's all right. Might have more to say once we've done the end-to-end testing.